tools/cpp/BUILD.windows.tpl

```diff
@@ -489,6 +492,72 @@ toolchain(
     toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
 )
 
+cc_toolchain(
+    name = "cc-compiler-arm64_windows-clang-cl",
+    toolchain_identifier = "clang_cl_arm64",
+    toolchain_config = ":clang_cl_arm64",
+    all_files = ":empty",
+    ar_files = ":empty",
+    as_files = ":clangcl_compiler_files",
+    compiler_files = ":clangcl_compiler_files",
+    dwp_files = ":empty",
+    linker_files = ":empty",
+    objcopy_files = ":empty",
+    strip_files = ":empty",
+    supports_param_files = 1,
+)
+
+cc_toolchain_config(
+    name = "clang_cl_arm64",
+    cpu = "arm64_windows",
+    compiler = "clang-cl",
+    host_system_name = "local",
+    target_system_name = "aarch64-pc-windows-msvc",
+    target_libc = "msvcrt",
+    abi_version = "local",
+    abi_libc_version = "local",
+    toolchain_identifier = "clang_cl_arm64",
+    msvc_env_tmp = "%{clang_cl_env_tmp}",
```

Comment:
Those env var values are also different for x86 and arm64, this is probably why the [tests are failing on Windows](https://storage.googleapis.com/bazel-untrusted-buildkite-artifacts/df9ece11-19b6-40d3-947d-4c5d10403786/src%5Ctest%5Cpy%5Cbazel%5Cbazel_windows_cpp_test%5Ctest.log). The second call of `_get_clang_cl_vars` will just override the result from the first call.

Reply:
Yes of course! Thanks a lot for pointing it out.